src/lightning_fabric/README.md

```diff
@@ -22,53 +22,182 @@ ______________________________________________________________________
 
 </div>
 
-## Maximum flexibility, minimum code changes
+# Lightning Fabric: Expert control.
```

Comment:
These changes are copied from the root README

Reply:
Hmm, I dont feel like this had to be changed here. That Fabric is fully ready for LLM is not correct imo.

Reply:
I'm fine Fine with it if it is requested

Reply:
This is just copy pasted verbatim from the root readme. Whatever we have in there, we should have were aswell